src/modules/funtions.js

```diff
@@ -0,0 +1,20 @@
+const Tasks = JSON.parse(localStorage.getItem('list')) || [];
+const input = document.querySelector('.inp');
+const Add = () => {
+  const create = {
+    description: input.value,
+    completed: false,
+    index: Tasks.length,
+  };
+  if (create.description !== '') {
+    Tasks.push(create);
+  }
+  localStorage.setItem('list', JSON.stringify(Tasks));
+};
+export default Add;
+
+const Remove = (index) => {
+  Tasks.splice(index, 1);
+  localStorage.setItem('list', JSON.stringify(Tasks));
+};
```

Comment:
- [ ] Your delete functionality is currently working. KIndly check that you are linking the remove event listener to the right element 👍🏽